diff --git a/rules/repos.bzl b/rules/repos.bzl
index b9ecdc2..c31bbc0 100644
--- a/rules/repos.bzl
+++ b/rules/repos.bzl
@@ -154,6 +154,7 @@ def coralnpu_repos2():
             "@coralnpu_hw//third_party/rules_hdl:0015-Use-short_path-for-python-runfiles-resolution.patch",
             "@coralnpu_hw//third_party/rules_hdl:0016-Add-V3AstNodeStmt-and-V3Dfg-gen-clone-cases-to-verilator.patch",
             "@coralnpu_hw//third_party/rules_hdl:0017-Clean-up-WAVES-env-var-to-avoid-spurious-traces.patch",
+            "@coralnpu_hw//third_party/rules_hdl:0018-Optional-SystemC-for-native-Cpp.patch",
         ],
         patch_args = ["-p1"],
     )
